src/main/java/stream/flarebot/flarebot/util/general/GeneralUtils.java

```diff
@@ -687,7 +687,7 @@ public static boolean canRunCommand(CommandType type, User user) {
             Guild g = Constants.getOfficialGuild();
 
             if (g.getMember(user) != null)
-                return g.getMember(user).getRoles().contains(g.getRoleById(type.getRoleId()));
+                return g.getMember(user).getRoles().contains(g.getRoleById(type.getRoleId())) || (FlareBot.instance().isTestBot() && g.getMember(user).getRoles().contains(Constants.getOfficialGuild().getRoleById(Constants.CONTRIBUTOR_ID)));
```

Comment:
This line is very long, format it or just spread it out a bit. Make it more readable.